core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteProcedure.java

```diff
@@ -86,14 +86,12 @@ protected boolean executeAction() throws SQLException, InterruptedException {
         boolean appendOutput = convertAppendOutput(getSqlProcedure().getValue());
 
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance()
+        Connection connection;
+        connection = ConnectionConfiguration.getInstance()
                 .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+                .orElseThrow(() -> new RuntimeException("Unknown connection name"));
```

Comment:
add 'connectionName' to the Exception message: cannot find connection 'connectionName'. This will help users during debugging

Reply:
done in next commit